go

```diff
@@ -34,7 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
-  exec_cmd "bundle exec rake test #{args.join ' '}"
```

Comment:
Uhh...all the tests I meant to write but never did? :stuck_out_tongue:  At any rate, how does `bundle exec rspec` treat you?